fix: update CLA link to correct URL (#20678)
pingcap/docs-cn: .github/pull_request_template.md

### First-time contributors' checklist <!--Remove this section if you're not a first-time contributor.-->

> - [ ] I've signed [**Contributor License Agreement**](https://cla.pingcap.net/pingcap/docs-cn) that's required for repo owners to accept my contribution.

Author: ```suggestion
- [ ] I've signed the [**Contributor License Agreement**](https://cla.pingcap.net/pingcap/docs), which is required for the repository owners to accept my contribution.
```
